examples/layer-browser/src/map.js

```diff
@@ -57,10 +60,20 @@ export default class Map extends PureComponent {
       clickedItem: null,
       queriedItems: null,
 
-      enableDepthPickOnClick: false
+      enableDepthPickOnClick: false,
+      metrics: null
     };
 
     this.deckRef = React.createRef();
+    this.metricsLoopHandle = null;
+  }
+
+  componentDidMount() {
+    this.metricsLoopHandle = window.requestAnimationFrame(this._metricsLoop);
```

Comment:
Use `_onMetrics`?

Reply:
Sure thing. I wasn't sure you wanted to keep `_onMetrics`.